### DISCUSSION TITLE
Ignore dataview code blocks in my daily notes

### DISCUSSION THREAD

**Discussion by tristanmanchester** [brianpetro/obsidian-smart-connections#357] [Q&A]
I have the following dataview code blocks in my daily notes:

```dataview
LIST
WHERE file.cday = this.file.day
```

to display notes created on that day, and

```dataview
LIST
WHERE file.mday = this.file.day
```

to display notes modified on that day. How can I ignore these parts with smart connections? Otherwise all my daily notes get connected.

**Comment by brianpetro** [comment]
If they're located under a common heading, then it's possible to exclude that heading in the settings.

🌴

**Reply by tristanmanchester** [reply]
They're under the headings `## Notes created today` and `## Notes modified today`, but when I put these headings in the `header_exclusions` bit in settings and force a refresh (or delete the jsons) it still matches them.

**Reply by tristanmanchester** [reply]
Okay I think I fixed it, I just put those two headings under another heading `# Previous Notes`.
